Divine/CLI/CommandLineActions.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Divine.Enums;
+using LSLib.LS;
+
+namespace Divine.CLI

```

Comment:
Although I realize that ConverterApp made the same mistake, keeping the lib and the tools under the same namespace (or at least the cli & gui app under the same namespace) in the future might be better.

Reply:
Why don't we just fix them all and instead of repeating the mistake?

Reply:
Agreed.